### PR TITLE
Use smallInstanceConfig() in Jet tests

### PR DESCRIPTION
**Pull request by ufukyilmaz**
This PR changes the instance configs of Jet tests to use `smallInstanceConfig()`. It aims to reduce the run time of these tests. 

Checklist:
- [x] Labels and Milestone set

### CODE REVIEW

**Comment by mmedenjak** [comment]
run-lab-run

**Comment by ufukyilmaz** [comment]
run-lab-run

**Review by gurbuzali — APPROVED** [review]

File: hazelcast/src/test/java/com/hazelcast/jet/core/metrics/MetricsTest.java [hunk 1]
```diff
@@ -361,6 +363,7 @@ public void metricsAreRestartedIfPipelineIsRunTwice() {
         new JobMetricsChecker(job).assertSummedMetricValue("total", 5);
     }
 
+    @Category(NightlyTest.class)
```
Comment (frant-hartm) [inline hazelcast/src/test/java/com/hazelcast/jet/core/metrics/MetricsTest.java:366]: Why the move to nightly? Does it take long?

File: hazelcast/src/test/java/com/hazelcast/jet/core/metrics/MetricsTest.java [hunk 2]
```diff
@@ -414,6 +417,7 @@ public void availableViaJmx() throws Exception {
         String instanceName = instance().getHazelcastInstance().getName();
         long sum = 0;
         int availableProcessors = Runtime.getRuntime().availableProcessors();
+        System.out.println("Available processors: " + availableProcessors);
```
Comment (frant-hartm) [inline hazelcast/src/test/java/com/hazelcast/jet/core/metrics/MetricsTest.java:420]: Leftover `System.out.println`, Either change to log or remove.

**Review by frant-hartm — CHANGES_REQUESTED** [review]

File: hazelcast/src/test/java/com/hazelcast/jet/pipeline/OrderedProcessingMultipleMemberTest.java [hunk 3]
```diff
@@ -88,15 +89,12 @@
...
-        int testCount = 16;
-        Config config = new Config();
-        for (int idx = 0; idx < testCount; idx++) {
-            EventJournalConfig eventJournalConfig = config
-                    .getMapConfig("test-map-" + idx)
-                    .getEventJournalConfig();
-            eventJournalConfig.setEnabled(true);
-            eventJournalConfig.setCapacity(30000); // 30000/271 ~= 111 item per partition
-        }
+        Config config = smallInstanceConfig();
```
Comment (ufukyilmaz) [inline hazelcast/src/test/java/com/hazelcast/jet/pipeline/OrderedProcessingMultipleMemberTest.java:92]: Although it extends a bit the scope of PR, I chose to convert this configuration setting  to wildcard here too

File: hazelcast/src/test/java/com/hazelcast/jet/core/metrics/MetricsTest.java (hunk 2 above)
Comment (ufukyilmaz) [reply hazelcast/src/test/java/com/hazelcast/jet/core/metrics/MetricsTest.java]: This changes in this class are left by mistake, removed them.

**Comment by ufukyilmaz** [comment]
run-lab-run

**Review by frant-hartm — APPROVED** [review]
Looks good now. Nice job @ufukyilmaz

**Comment by mmedenjak** [comment]
run-lab-run

**Review by frant-hartm — APPROVED** [review]

**Comment by ufukyilmaz** [comment]
run-lab-run

**Comment by mmedenjak** [comment]
Merging as the test failures are unrelated. We'll gradually improve the PR builder and once it becomes fast and stable again, we can revert back to requiring a clean pass.

**Pull request merged**